src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage/Forms/form_seo.html.twig

```diff
@@ -72,20 +72,21 @@
   <div class="row">
     <div class="col-md-9">
       <div class="alert alert-info" role="alert">
+        {% set metaUrl = path('admin_meta') ~ '#meta_fieldset_general' %}
```

Comment:
I think the anchor needs to be changed as well, I don't see it in the new SF page as the form has changed
unfortunately the form has no ID for now so either use the ID of an input `form_set_up_urls_friendly_url_0` for example..
But it's a bit dirty I think it would be better to assign an ID to the form

Reply:
I've noticed that too, but didn't want to address this anchor bug "here".